libbindgen/src/ir/enum_ty.rs

```diff
@@ -49,8 +49,11 @@ impl Enum {
         }
 
         let declaration = ty.declaration().canonical();
-        let repr = Item::from_ty(&declaration.enum_type(), None, None, ctx)
-            .ok();
+        let et = declaration.enum_type();
+        if et.is_none() {
+            return Err(ParseError::Continue);
+        }
+        let repr = Item::from_ty(&et.unwrap(), None, None, ctx).ok();
```

Comment:
@Wafflespeanut sorry, I completely misunderstood suggestions from @emilio earlier. I'll work on this and let you know if I have any questions.

By the way, running `cargo test --features llvm_stable`  shows no failures on my machine - the only way I get to see failed checks is by updating this PR. Do you have any idea why that might be?

Thanks for all your patience and guidance, I'll be a rustacean yet ;)